modules/core/src/main/java/org/apache/synapse/config/SynapseConfiguration.java

```diff
@@ -1447,10 +1452,10 @@ public synchronized void destroy(boolean preserverState) {
         }
 
         //destroy inbound endpoint
-		for (InboundEndpoint endpoint : getInboundEndpoints()) {
-			endpoint.destroy();
-		}         
-        
+        for (InboundEndpoint endpoint : getInboundEndpoints()) {
```

Comment:
Shall we explain the purpose of this code segment in a comment?

Reply:
Updated